mirage/scenarios/default.ts

```diff
@@ -105,7 +105,7 @@ function registrationScenario(
         id: '1',
         primaryColor: 'green',
         secondaryColor: 'blue',
-        navbarLogoImage: 'http://somelogoimageurl',
+        navbarLogoImage: 'https://seeklogo.net/wp-content/uploads/2017/04/New-Google-Earth-logo.png',
```

Comment:
we have a [`placekitten` util](https://github.com/robustdj/ember-osf-web/blob/add3ef17747ac7dfccc64ecf80a94a6189ece9b6/mirage/utils.ts#L3-L5) you can use for this, if you'd like